.ansible-lint

```diff
@@ -0,0 +1,21 @@
+exclude_paths:
+    - mkdocs.yml
+    - docs/
+    - tests/config/
+    - tests/secrets.sample.yaml
+    - tests/vars.sample.yaml
+skip_list:
+    - var-naming[no-role-prefix]
+    - yaml[line-length]
+    - risky-shell-pipe # we have pipefail set in common_defaults shell_header:
+    - name[casing] # do we want that one for consistency? maybe?
+    - no-changed-when # any point adding changed_when: false in all our shell?
+    - command-instead-of-shell
+    - latest[git] # yes we want to clone latest of install_yamls
+strict: true
```

Comment:
i think this is what causes the error in the current run
ansible-lint passes but the warning are fatal because of strict: true

        *   -s, --strict          Return non-zero exit code on warnings as well as errors

https://pipelines.actions.githubusercontent.com/lxduM8vjEqYXo2nftVdE1YENbWz134qiZSWuWpyOCy4dXvdVO4/_apis/pipelines/1/runs/194/signedlogcontent/2?urlExpires=2023-09-25T07%3A12%3A27.9993435Z&urlSigningMethod=HMACV1&urlSignature=8ioUhSu80P%2Fd1F0dvC3CXRMI2vdQ%2FWBmTbhJ58iGijs%3D


```
2023-09-22T13:57:15.8196602Z Failed: 0 failure(s), 6 warning(s) on 76 files. Last profile that met the validation criteria was 'min'.
2023-09-22T13:57:15.8197280Z You are using a pre-release version of ansible-lint.
2023-09-22T13:57:15.9233112Z ##[error]Process completed with exit code 2.
```